contributing/publishing.md

```diff
@@ -22,38 +22,44 @@ For more information about publishing take a look at:
 
 ## Verifying the Change Log
 
-One of the members of [Doc Maintainers](https://github.com/orgs/forcedotcom/teams/doc-maintainers/members) would review the changelog and make any changes to the release branch. A pull request will be opened against the release branch with updates to be included.
+One of the members of [Doc Maintainers](https://github.com/orgs/forcedotcom/teams/doc-maintainers/members) will review the changelog and make any changes to the release branch. A pull request will be opened against the release branch with updates to be included.
 
 ## Merging the Release Branch into Main
 
-After the change log has been approved and merged into your release branch, it's time to prepare main with the new changes for the publish. A Github Action workflow is executed to merge the release branch. We are specifically using the rebase strategy because we want all the commits from our release branch to be applied on top of the commits in main.
+After the change log has been approved and merged into your release branch, it's time to prepare the `main` branch with the new changes for the publish. A GitHub Action workflow is executed to merge the release branch. We are specifically using the rebase strategy because we want all the commits from our release branch to be applied on top of the commits in the `main` branch.
 
 ### To run the merge process:
 
-1. From the github repository navigate to the Action Tab and Select the PreRelease workflow on the left
-1. Click the 'Run Workflow' button and input the branch to be merged into the input box.
-1. Click the `Run Workflow` button.
+1. From the GitHub repository navigate to the Action Tab, and Select the PreRelease workflow on the left
+1. Click the 'Run Workflow' dropdown button on the right
+1. In the form that appears, set the branch to `develop`, and set the 'branch to be released' input box to the name of the release (eg `release/v56.0.0`)
+1. Click the 'Run Workflow' button.
 
-The PreRelease job will verify if the version of the branch to be merged is newer than what is currently in main and update main with the release branch.
+The PreRelease job will verify if the version of the branch to be merged is newer than what is currently in the `main` branch and update `main` with the release branch.
 
 ## Publishing Main
 
-The merge into main will trigger a run of the 'Test, Build, and Release' GHA workflow that will:
+The merge into `main` will trigger a run of the 'Test, Build, and Release' GHA workflow (https://github.com/forcedotcom/salesforcedx-vscode/actions/workflows/testBuildAndRelease.yml) that will:
 - run the tests
 - build vsix files
 - send a slack notification that a release workflow has been initiated
-- create a tag and release in Github
+- create a tag and release in GitHub
+
+In the 'Test, Build, and Release' workflow:
```

Comment:
Should this come after line 62, for the instructions on how to "...approve the publish job"?

Reply:
heh...we actually have it both the Test, Build <pause>, and Release job and the publish job
We probably don't need the pause in the build job b/c I can't envision a time when you'd want to abort after the tests/build happens but pre release/tag/merge.  I left it in partially due to just being overly careful with the process working as expected. 

At this point I could just take it out.  thoughts?